### PR TITLE
Support to collect queries from http basic auth enabled cluster

### PR DESCRIPTION
**Pull request by ashishtadose**

### CODE REVIEW

**Comment by romanvainbrand** [comment]
Hi Ashish,
Thanks for your contribution.
I wonder if we add params such as username and password to get wouldn't it be better create a class which will get all params in __init__ and all the get calls will stay clean.

@ashishtadose @guy
WDYT?

File: analyzer/collect.py [hunk 1]
```diff
@@ -76,7 +84,7 @@ def main():
             time.sleep(args.delay)  # for rate-limiting
             log.info("Downloading {} -> {}", url, output_file)
             try:
-                response = get(url)
+                response = get(endpoint, args.username, args.password)
```
Comment (sbernauer) [inline analyzer/collect.py:87]: I tested out for patch. Needed quite some time to find the problem. This should be
`response = get(url, args.username, args.password)`

**Review by sbernauer — CHANGES_REQUESTED** [review]
Hi @ashishtadose thanks for you patch! I did notice a small problem (see Review)
@ashishtadose and @guy feedback regarding @romanvainbrand's comment would be great!

**Comment by Guy-Mast** [comment]
Hi @sbernauer and all,
I will work on incorporating all the inputs shared in this PR and thread and will provide an updated PR.

**Comment by Guy-Mast** [comment]
all,
please see the new PR https://github.com/varadaio/presto-workload-analyzer/pull/5 which is based on @ashishtadose work here and takes into consideration the follow-up feedbacks we got.

please let me know if you have any comments.

**Pull request closed without merging**